airgun/views/common.py

```diff
@@ -39,9 +40,9 @@ class BaseLoggedInView(View):
     taxonomies = ContextSelector()
     flash = SatFlashMessages()
     validations = ValidationErrors()
-    current_user = Text("//li[@id='account_menu']")
-    account_menu = Text("//li[@id='account_menu']")
     logout = Text("//a[@href='/users/logout']")
+    current_user = Dropdown('OUIA-Generated-Dropdown-1')
+    account_menu = Dropdown('OUIA-Generated-Dropdown-1')
```

Comment:
Why do we have two widgets with the same locator? I see this isn't unique to your use - that's how it was before too.

I don't want to blow up the scope of your PR here, but that's an obvious TODO/airgun issue - remove one of these widgets, and update all uses of the removed one to instead use the remaining widget.

Reply:
https://github.com/SatelliteQE/airgun/issues/550

Reply:
I read my own mind.